### ISSUE TITLE
Please release new npm package 

### ISSUE THREAD

**Issue by greglobinski** [zabute/gatsby-plugin-svgr#17]
Please release new npm package for the plugin,  compliant with the last [commit](https://github.com/zabute/gatsby-plugin-svgr/commit/9808d913eccd488229b379deeb2bbd27656ce9bc#diff-b9cfc7f2cdf78a7f4b91a753d10865a2). The version 1.0.0 is trying to install [svgr](https://www.npmjs.com/package/svgr) which is deprecated

I tried to install `gatsby-plugin-svgr` on a fresh installation of the default [gatsby-starter-blog](https://github.com/gatsbyjs/gatsby-starter-blog) and got this

```
λ yarn add gatsby-plugin-svgr
yarn add v1.3.2
warning ..\..\..\package.json: No license field
[1/4] Resolving packages...
warning gatsby-plugin-svgr > svgr@2.0.0: Please install @svgr/cli, @svgr/core or @svgr/webpack
[2/4] Fetching packages...
info fsevents@1.2.4: The platform "win32" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "gatsby > friendly-errors-webpack-plugin@1.7.0" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
warning "gatsby > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
warning "gatsby-plugin-react-helmet > react-helmet@5.2.0" has unmet peer dependency "react@>=15.0.0".
warning "gatsby-plugin-react-helmet > react-helmet > react-side-effect@1.1.5" has unmet peer dependency "react@^0.13.0 || ^0.14.0 || ^15.0.0 || ^16.0.0".
warning "gatsby-plugin-typography > react-typography@0.16.13" has unmet peer dependency "react@^0.14.0 || ^15.0.0 || ^16.0.0".
warning "gatsby-transformer-remark > graphql-type-json@0.1.4" has unmet peer dependency "graphql@>=0.4.0".
[4/4] Building fresh packages...
[1/1] ⢀ svgr
[-/1] ⠠ waiting...
[-/1] ⠠ waiting...
[-/1] ⠠ waiting...
error C:\Users\Greg\_dev\tests\test-for-svgr-2\node_modules\svgr: Command failed.
Exit code: 1
Command: node message.js; sleep 10; exit 1;
Arguments:
Directory: C:\Users\Greg\_dev\tests\test-for-svgr-2\node_modules\svgr
Output:
module.js:549
    throw err;
    ^

Error: Cannot find module 'C:\Users\Greg\_dev\tests\test-for-svgr-2\node_modules\svgr\message.js;'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
```

**Comment by kremalicious** [comment]
There might be more changes required to get full SVGR 2.0.0 compatibility I think, releasing a new package right now might break some projects. While #16 makes the plugin install successfully, actually using it yields errors for every imported SVG file:

```bash
Loader ./node_modules/@svgr/webpack/lib/index.js?{} didn't return a function
```

Anyone can confirm this within their own projects? If so, I would create a new issue for this

To reproduce:

```bash
gatsby new gatsby-starter-blog https://github.com/gatsbyjs/gatsby-starter-blog
cd gatsby-starter-blog/

npm i zabute/gatsby-plugin-svgr
#... add plugin to gatsby-config.js, import some sample SVG in the project
npm run dev
```

**Comment by kremalicious** [comment]
As a quick temporary workaround, I rolled back and locked the SVGR version in my fork in a branch called `locksvgr`. To use, install it with:

```bash
npm i kremalicious/gatsby-plugin-svgr#locksvgr
```

or add this to your `package.json`:

```json
"gatsby-plugin-svgr": "github:kremalicious/gatsby-plugin-svgr#locksvgr",
```

**Comment by zabute** [comment]
I'm seeing that error too. That's why I haven't published #16 yet.

Also tried `@svgr/webpack` in a non-gatsby project with webpack 1 and it fails with the same error. I'm inclined to think this is an issue with the loader but I'm not sure. Until I figure it out, I have released the patch from @kremalicious's fork.

**Comment by greglobinski** [comment]
Ok, I understand.
